cmd/kops/create_cluster.go

```diff
@@ -389,8 +389,9 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		}
 
 		for _, etcdCluster := range cloudup.EtcdClusters {
-			etcd := &api.EtcdClusterSpec{}
+			etcd := api.EtcdClusterSpec{}
```

Comment:
I thought it was because apimachinery was unhappy with me for it.  But I may have been doing something else wrong as well...